test/cucumber/steps/AddEvaluation.groovy

```diff
@@ -0,0 +1,178 @@
+package steps
+
```

Comment:
(OK) - Test behavior should strictly conform to scenario semantics
(OK) - Avoid ambiguities due to similar step sentences
(OK) - Do not duplicate test code
(OK) - Test should clean up environment at the end
(OK) - Do not mix GUI and controller tests
(OK) - Tests should be platform and language independent